src/Entry/EGL/EGLRuntime.cpp

```diff
@@ -334,7 +334,7 @@ void getMaxSurfaceSize(int resolutionLevel, float* maxSurfaceSize)
   }
 }
 
-std::tuple<std::string, std::map<int, std::vector<char>>> render(
+std::tuple<std::string, std::vector<std::pair<std::string, std::vector<char>>>> render(
```

Comment:
Why not use std::map<std::string, std::vector<char>>? Is the key not unique in a single render process?

Reply:
Key might not be unique is one of the reasons. These two forms are equivalent though, don't pay the overhead if you don't need especially in API level. If the caller need the map, just recollects the vector into map. Prefer vector than map in any possible cases unless map is necessary.